### DISCUSSION TITLE
From poetry back to pip-tools

### DISCUSSION THREAD

**Discussion by guettli** [python-poetry/poetry#4926] [General]
Here is a twitter thread where some people say the switch back from poetry to pip-tools.

https://twitter.com/uninen/status/1468674427835625484

Maybe this is a good time to ask them about detailed negative feedback, so that you can improve poetry.

**Comment by finswimmer** [comment]
Hello @guettli,

thanks for sharing. I believe most of the people commenting their, doesn't get the main goal of poetry: Painless **development** of python packages. So the focus is on reproducible python environment, building and publishing packages. 

What a lot of people miss, because they hope/expect to get it with poetry as well, is the topic deployment, especially in conjunction with docker. This was never a goal by poetry until now. It is often asked and seems to be useful. So I'm confident we will focus on this topic somewhere in the future. You can already do a lot out-of-the-box with poetry, but of course not that smooth like with tools which focus on that.

fin swimmer
